velox/vector/BaseVector.h

```diff
@@ -430,6 +430,11 @@ class BaseVector {
     vector_size_t sourceIndex;
     vector_size_t targetIndex;
     vector_size_t count;
+
+    bool mergeable(const CopyRange& next) const {
```

Comment:
Would it be possible to add documentation and test for this new API?

Reply:
I will add some document.  This is probably too trivial to be tested individually.